src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json

```diff
@@ -4083,14 +4238,213 @@
+        "for": "0m",
+        "frequency": "1h",
+        "handler": 1,
+        "message": "${ServicingJobs} servicing jobs were executed in R&D queues and need investigation\n\nFor details on how servicing jobs are redirected, go to https://github.com/dotnet/core-eng/blob/main/Documentation/Project-Docs/ServicingJobRedirection/DesignDocs.md",
+        "name": "Servicing jobs in R&D queues alert",
+        "noDataState": "keep_state",
+        "notifications": [
+          {
+            "uid": "statusHook"
+          }
+        ]
+      },
+      "aliasColors": {},
+      "bars": true,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "Kusto-engsrv",
+      "decimals": 0,
+      "description": "Shows the number of servicing jobs executed in R&D queues",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {}
+        },
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 8,
+        "w": 7,
+        "x": 0,
+        "y": 40
+      },
+      "hiddenSeries": false,
+      "id": 62,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": false,
+        "total": false,
+        "values": false
+      },
+      "lines": false,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "7.4.3",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "database": "engineeringdata",
+          "expression": {
+            "groupBy": {
+              "expressions": [],
+              "type": "and"
+            },
+            "reduce": {
+              "expressions": [],
+              "type": "and"
+            },
+            "where": {
+              "expressions": [],
+              "type": "and"
+            }
+          },
+          "hide": false,
+          "pluginVersion": "3.3.2",
+          "query": "let UntrackedQueues = Jobs \n| project QueueName = tolower(QueueName)\n| where QueueName contains \"osx\" or QueueName contains \"perf\" or QueueName contains 'arm' or QueueName contains \"arcade\" or QueueName contains \"xaml\" or QueueName contains \"appcompat\" or QueueName contains \"iot\"\n| distinct QueueName;\nJobs\n| where  $__timeFilter(Queued)\n| where tolower(QueueName) !in (UntrackedQueues)\n| extend targetBranch=parse_json(Properties)[\"System.PullRequest.TargetBranch\"]\n| make-series ServicingJobs=countif((Branch contains \"release/\" or targetBranch contains \"release/\") and QueueName !endswith \".svc\") default=0 on Queued from $__timeFrom  to $__timeTo step 1h\n| mv-expand Queued, ServicingJobs\n| project toint(ServicingJobs), todatetime(Queued)\n| order by Queued",
```

Comment:
The logic here is a bit different than in Helix API. This query can in theory report some false positives for branches names like `not-a-release/abc` although I'm not sure if we actually have cases like this.

And after writing the above I wonder if we shouldn't add this dashboard in `dotnet-helix-service` instead? That would allow us to keep redirection logic and monitoring in the same repo.

Reply:
It definitely should be in helix, since it's all about helix data and helix behavior.